samples/GUITests.robot

```diff
@@ -0,0 +1,124 @@
+*** Settings ***
+Documentation   Zoomba GUI Library Tests
+Library         Zoomba.GUILibrary
+
+*** Variables ***
+${browser}     chrome
+
+*** Keywords ***
+Test Case Setup
+    [Arguments]    ${url}=https://github.com/      ${browser}=${browser}
+    Open Browser   ${url}    browser=${browser}
+    Maximize Browser Window
+    Set Selenium Speed    0.2s
+
+*** Test Cases ***
+Wait for Keywords Test
```

Comment:
Since you're using [Teardown] here, it might be more visually appealing to move your Test Case Setup to [Setup] for each of these test cases. I was going to suggest moving it all to Test Setup in *** Settings ***, but since the URL parameter is different for many I don't think that's possible.